IfcElementAssembly Rail Import BIMQ 2021-07-21T07:37:32.8695288Z (#185)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/i/IsAccessibleByVehicle_3R5N45JWj9UumSu_fUKeWz/Documentation.md

> Is the banquet element accessible by vehicle (TRUE) or not (FALSE). (type of accessibility where applicable)

Reviewer: ```suggestion
Indicates whether the element is accessible by a vehicle or not.
```
